src/pyetm/services/scenario_runners/update_saved_scenario.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, Dict
+from pyetm.services.scenario_runners.base_runner import BaseRunner
+from ..service_result import ServiceResult
+from pyetm.clients.base_client import BaseClient
+
+
+class UpdateSavedScenarioRunner(BaseRunner[Dict[str, Any]]):
+    """
+    Runner for updating a SavedScenario in MyETM.
+
+    PUT /api/v3/saved_scenarios/:id
+
+    Args:
+        client: The HTTP client to use
+        saved_scenario_id: ID of the SavedScenario to update
+        update_data: Dictionary with fields to update (title, description, private, discarded)
+        **kwargs: Additional arguments passed to the request
+    """
+
+    ALLOWED_KEYS = ["title", "description", "private", "discarded"]
```

Comment:
Updating is not in scope at this moment, but for History we will need to be able to update the scenario ID (save a new version).

```suggestion
    ALLOWED_KEYS = ["title", "scenario_id", "private", "discarded"]
```

I'd suggest not to include the description. There is not really a use case for it, and with Action Text you can do a lot more then through the API.